1/README.md

```diff
@@ -40,9 +40,9 @@ C4 is meant to provide a reusable optimal collaboration model for open source so
 
 ### 2.2. Licensing and Ownership
 
-1. The project SHALL use a share-alike license such as the MPLv2, or a GPLv3 variant thereof (GPL, LGPL, AGPL).
+1. The project SHALL use a license such as Apache 2, MIT, MPLv2, or a GPLv3 variant thereof (GPL, LGPL, AGPL).
```

Comment:
I guess these changes make this document the BigchainDB variant of the C4? Maybe we could indicate that in the title, just so it's clear that it's not the same as the original/standard C4? There could even be a "diff document" listing the differences between the two.

As for the license, I think Bruce's current thinking is that all BigchainDB software will be licensed Apache v2 for now. Not other licenses.

Reply:
> I guess these changes make this document the BigchainDB variant of the C4? Maybe we could indicate that in the title, just so it's clear that it's not the same as the original/standard C4? There could even be a "diff document" listing the differences between the two.

Yes, this is our variant. [unprotocols.org 1/C4](https://github.com/unprotocols/rfc/blob/master/1/README.md) says _This RFC is equivalent (with the exception of minor cosmetic changes) to ZeroMQ RFC 42/C4_. I can say that this C4 is a customized version of the one by unprotocols. About the diff, users can do it themselves, it's really easy and we have one less document to maintain.

> As for the license, I think Bruce's current thinking is that all BigchainDB software will be licensed Apache v2 for now. Not other licenses.

I was keeping it open in case we start other projects with different requirements.

Reply:
We can always modify this document in the future, if we decide to license some projects with a license other than Apache v2.